packages/quantic/force-app/main/translations/fr.translation-meta.xml

```diff
@@ -284,4 +284,120 @@
         <label> :</label>
         <name>quantic_Colon</name>
     </customLabels>
+    <customLabels>
```

Comment:
@jpmarceau Please review this file. Most of the strings were copied from atomic.